.dotnet/tests/Embeddings/EmbeddingsMockTests.cs

```diff
@@ -95,9 +104,18 @@ public async Task GenerateEmbeddingsWithStringsDeserializesUsage()
     public async Task GenerateEmbeddingsWithStringsDeserializesVector()
     {
         string base64Embedding = Convert.ToBase64String([
-            0x00, 0x00, 0x80, 0x3F, // 1.0f
-            0x00, 0x00, 0x00, 0x40, // 2.0f
-            0x00, 0x00, 0x40, 0x40  // 3.0f
+            0x00,
+            0x00,
+            0x80,
+            0x3F, // 1.0f
```

Comment:
I think this is worth allowing. Thoughts?

Reply:
Yuck. Hmm, maybe we're thinking about this the wrong way, and the rule is just making the smell a bit more pungent -- could we do something like this, instead?

```csharp
string base64Embedding = Convert.ToBase64String(
[
    ..BitConverter.GetBytes(1.0f),
    ..BitConverter.GetBytes(2.0f),
    ..BitConverter.GetBytes(3.0f),
]);
```

Reply:
I'll opt in to kick the can down the road just to get the 99% good stuff merged.